rocketry/session.py

```diff
@@ -62,35 +64,38 @@ class Config:
 
     multilaunch: bool = False
     func_run_id: Callable = uuid
-    max_process_count = cpu_count()
+    max_process_count:int = cpu_count()
     tasks_as_daemon: bool = True
     restarting: str = 'replace'
     instant_shutdown: bool = False
 
     timeout: datetime.timedelta = datetime.timedelta(minutes=30)
     shut_cond: Optional['BaseCondition'] = None
-    cls_lock: Type = threading.Lock
+    cls_lock: threading.Lock = threading.Lock
```

Comment:
I think semantically this is a ``Type`` (should be __lock like__ type, perhaps in the future we allow other types of locking to support parallel scheduling instances). Or were there perhaps problems with this line?

Reply:
This definitely should be Type imo, but when I try and execute the code I seem to get a validation error when setting cls_lock:
```
Exception has occurred: ValidationError
1 validation error for Config
cls_lock
  Input should be a type [type=is_type, input_value=<built-in function allocate_lock>, input_type=builtin_function_or_method]
```

Type[threading.Lock] seems to throw a similar error. The typing I've set here isn't ideal and still should be changed though. I've changed the type to be `Callable` since from the error in thinks I'm passing in a function type if that's okay?